doc/riverctl.1.scd

```diff
@@ -272,6 +273,8 @@ matches everything while _\*\*_ and the empty string are invalid.
 	  and existing views.
 	- *csd*: Use client-side decorations for the view. Applies to new
 	  and existing views.
+	- *tag*: Set the initial tags of the view. Requires the tags as
+	  an argument.
```

Comment:
```suggestion
	  an argument. Applies only to new views.
```

Also, there's an example table from `list-rules` further down that should probably have its column order changed to match the new output.